src/org/gk/qualityCheck/HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities.java

```diff
@@ -0,0 +1,102 @@
+package org.gk.qualityCheck;
+
+import org.gk.database.InstanceListPane;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.GKSchemaClass;
+
+import java.util.*;
+
+/**
+ * Flags human ReactionlikeEvents that do not have a populated 'disease' attribute, but have non-human participants.
+ */
+
+public class HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities extends AbstractQualityCheck {
+
+
+    @Override
+    public QAReport checkInCommand() throws Exception {
+        QAReport report = new QAReport();
+        MySQLAdaptor dba = (MySQLAdaptor) dataSource;
+        QACheckUtilities.setSkipList(Arrays.asList("168249")); // Innate Immunity Pathway
```

Comment:
Could the db id for Innate Immunity Pathway be put into a named constant rather than a comment so it is part of the code?  I worry about the comment being de-synced from the value.